projects/kit/components/tabs/tabs.directive.ts

```diff
@@ -55,13 +62,17 @@ export class TuiTabsDirective implements AfterViewChecked {
     }
 
     ngAfterViewChecked(): void {
-        const {tabs, activeElement} = this;
+        timer(0)
```

Comment:
Let's not introduce changes to this component, it's rather fundamental.

Reply:
I don't why, but when I navigate to another page in `ngAfterViewChecked` we have previous cached `[tuiTab]` elements